src/activities/CreateLayerFromArcGisServerUrl.ts

```diff
@@ -0,0 +1,50 @@
+import type { IActivityHandler } from "@geocortex/workflow/runtime";
+import Layer from "@arcgis/core/layers/Layer";
+
+interface CreateLayerFromArcGisServerUrlInputs {
+    /**
+     * @displayName URL
+     * @description The URL to the ArcGIS Server service.
+     * @required
+     */
+    url: string;
+
+    /**
+     * @description Any of the layer's properties for constructing the layer instance (e.g. popupTemplate, renderer, etc.).
+     */
+    properties?: any;
```

Comment:
Do layer properties have a type?

Reply:
Not in the type definitions for the method we are using
![image](https://user-images.githubusercontent.com/5251278/217033834-e4331e3d-bf7e-45b1-bf25-732272fc8e37.png)

That said, that method does just call the correct layer constructor, so there are a handful of interfaces, like `FeatureLayerProperties`, `ArcGISCachedServiceProperties`, `ArcGISImageServiceProperties`, `ArcGISMapServiceProperties`, etc., that must be used in practice. The trouble is that there are so many of them and they each have loads of properties. We would have to supply the superset of all possibilities. At which point, it is basically `any`.

Reply:
I figured it must be something like that